internal/wasm/module_test.go

```diff
@@ -68,7 +68,7 @@ func TestExternTypeName(t *testing.T) {
 	}{
 		{"func", ExternTypeFunc, "func"},
 		{"table", ExternTypeTable, "table"},
-		{"mem", ExternTypeMemory, "mem"},
+		{"mem", ExternTypeMemory, "memory"},
```

Comment:
this is the only impact and except in text parsing, this is only used for late errors on invalid memory